docs/REVIEWING.md

```diff
@@ -36,6 +36,40 @@ following items should also be checked:
 - [ ] Does it comply with [component spec](specs/component.md)?
 - [ ] Does it comply with the [instrumentation spec](specs/instrumentation.md)?
 
+
+### Sink review checklist
+
+This checklist is specific for Vector's sink code.
+
+## Logic
```

Comment:
The heading levels seem off- the top one is 3 , then the subs are 2 ? Shouldn't it be the other way around?